qiita_db/test/test_util.py

```diff
@@ -1309,19 +1309,21 @@ def test_quick_mounts_purge(self):
 
 class ResourceAllocationPlotTests(TestCase):
     def setUp(self):
-
-        self.PATH_TO_DATA = ('./qiita_db/test/test_data/'
-                             'jobs_2024-02-21.tsv.gz')
-        self.CNAME = "Validate"
-        self.SNAME = "Diversity types - alpha_vector"
+        self.CNAME = "Split libraries FASTQ"
+        self.SNAME = "QIIMEq2"
         self.col_name = 'samples * columns'
-        self.df = pd.read_csv(self.PATH_TO_DATA, sep='\t',
-                              dtype={'extra_info': str})
+        self.columns = [
+                "sName", "sVersion", "cID", "cName", "processing_job_id",
+                "parameters", "samples", "columns", "input_size", "extra_info",
+                "MaxRSSRaw", "ElapsedRaw"]
+        self.df = pd.DataFrame(
+            qdb.util._retrieve_resource_data(self.CNAME, self.SNAME),
+            columns=self.columns)
```

Comment:
IMOO _retrieve_resource_data should just assign the column names and create the dataframe so they are always the same. The current separations feels clunky as the column. In other words, this should only be:
`self.df = qdb.util._retrieve_resource_data(self.CNAME, self.SNAME)`.